docs/reference/API.md

```diff
@@ -55,104 +63,163 @@ The `schema` boolean will determine if this database is created with
 an empty schema, or if it is running in "schema free" mode. It
 defaults to false.
 
+### Example:
+
+Create a database with the following:
+organization: admin
+dbid: owl_db
+label: label
+comment "information about owls"
+
+The payload in this case is: 
+ ```jsx
+{  
+   "prefixes": {"@type": "doc_prefix", 
+              "@schema": "schema_prefix"}, 
+   "comment" : "information about owls",
+   "label" : "owl label" 
```

Comment:
Unfortunate example, I used owl the animal as an example (information about owls). Will change it to cows instead, to avoid confusion. To the best of my knowledge the Web Ontology Language (OWL) is capitalised?